apps/shell/app.js

```diff
@@ -1,14 +1,18 @@
-var fs        = require('fs');
-var http      = require('http');
-var path      = require('path');
-var WebSocket = require('ws');
-var express   = require('express');
-var pty       = require('node-pty');
-var hbs       = require('hbs');
-var dotenv    = require('dotenv');
-var Tokens    = require('csrf');
-var url       = require('url');
-var port      = 3000;
+const fs        = require('fs');
+const http      = require('http');
+const path      = require('path');
+const WebSocket = require('ws');
+const express   = require('express');
+const pty       = require('node-pty');
+const hbs       = require('hbs');
+const dotenv    = require('dotenv');
+const Tokens    = require('csrf');
+const url       = require('url');
+const yaml      = require('js-yaml');
+const glob      = require("glob");
+const port      = 3000;
+const log       = console.log;
```

Comment:
This is one change I would undo `const log       = console.log;`. Everyone looking at JavaScript code knows what `console.log` does, but seeing the short form `log(` might surprise people and because of this, even though it is shorter it reduces readability.